rmf_robot_sim_common/include/rmf_robot_sim_common/slotcar_common.hpp

```diff
@@ -130,6 +130,13 @@ enum class SteeringType
 class SlotcarCommon
 {
 public:
+  typedef struct UpdateResult
```

Comment:
Nit: Just `struct` would be alright, we should move towards modern C++ with the rest of the RMF codebase